### DISCUSSION TITLE
希望开荒能增加一个弹药不足了使用另外一队的功能

### DISCUSSION THREAD

**Discussion by Undeaduno** [LmeSzinc/AzurLaneAutoScript#2372] [Ideas]
三星成就靠一队打完道中真的很难啊

**Comment by ec7529a2** [comment]
一队打不完道中一般是你配队有问题，建议去wiki上看点攻略，配一个或者抄一个好点的道中队。
如果你缺船，配不出好队伍，可以选择先打到绿海再打三星。
再不行手操，反正一张图只要打一次三星。
